test/regression-evm/test-and0.sh.out.golden

```diff
@@ -0,0 +1,240 @@
+/* Fl Fn D Sfa Cl */
```

Comment:
Could you link to the original `evm-semantics` `krun` tests you added (like this one)? I want to be sure the expected output is correct.

Reply:
Not sure I understand what you're asking for.

Reply:
I mean, I'd like to look at the original tests from the `evm-semantics` repo.

Reply:
The outputs for the `krun` tests are these final configs, and I want to be sure that they correspond to the expected outputs in the `evm-semantics` repository.

Reply:
The expected output for these tests is https://github.com/kframework/evm-semantics/blob/master/tests/templates/output-success-haskell.json.